utils/install_dag.sh

```diff
@@ -33,6 +53,9 @@ AIRFLOW_DAG_PATH=${AIRFLOW_HOME}/dags/${IMAGE}
 CONTAINER_DAG_PATH=/dags
 POST_INSTALL=${AIRFLOW_DAG_PATH}/post_install.sh
 
+
+is_docker_daemon_up
```

Comment:
@smpiano I don't like the implementation since you can't tell that inside `is_docker_daemon_up` I would:
a) Call the method `wait_for_docker_daemon`
b) move the loop outside so it can read:
```
while !is_docker_daemon_up; 
done
```

Reply:
@enriquetuya PR is updated taking into account the comments. Could you please take a look?